src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/TilkjentYtelseService.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.familie.ks.sak.kjerne.beregning
+
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.Vilkårsvurdering
+import no.nav.familie.ks.sak.kjerne.beregning.domene.TilkjentYtelse
+import no.nav.familie.ks.sak.kjerne.beregning.endretUtbetaling.AndelTilkjentYtelseMedEndretUtbetalingBehandler
+import no.nav.familie.ks.sak.kjerne.beregning.regelverkFørFebruar2025.gammel.RegelverkFørFebruar2025AndelGeneratorGammel
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonopplysningGrunnlag
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class TilkjentYtelseService {
+    fun beregnTilkjentYtelse(
+        vilkårsvurdering: Vilkårsvurdering,
+        personopplysningGrunnlag: PersonopplysningGrunnlag,
+        endretUtbetalingAndeler: List<EndretUtbetalingAndelMedAndelerTilkjentYtelse> = emptyList(),
+    ): TilkjentYtelse {
+        val tilkjentYtelse =
+            TilkjentYtelse(
+                behandling = vilkårsvurdering.behandling,
+                opprettetDato = LocalDate.now(),
+                endretDato = LocalDate.now(),
```

Comment:
Det er nok en fordel å passere inn et `Clock` objekt i konstruktøren og passere det inn som et parameter her `LocalDate.now(clock)`. Jeg har allerede definert en `Clock` bønne her https://github.com/navikt/familie-ks-sak/blob/main/src/main/kotlin/no/nav/familie/ks/sak/config/ClockConfig.kt 

Fordel med det er at man:
1. Kan skrive enhetstester uten (static) mocks
2. Muliggjør "tidsreising" i applikasjonen da man kan starte applikasjon med en egendefinert `Clock` bønne

Reply:
Ser at i enkelte andre servicer er `LocalDateProvider` brukt. Burde vi holde oss til én fremgangsmåte, så det ikke blir to måter å definere/mocke tid på?

Reply:
Vet ikke om jeg har så lyst til å gjøre noen av disse endringene i en omstrukturerings-PR som skulle være en rask ikke alt for omfattende greie, men er enig i at det er lurt, for testbarheten, å ta ibruk `Clock` eller `LocalDateProvider` 👍